com.milesalan.mepo.metainfo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component>
+  <id>com.milesalan.mepo</id>
+  <metadata_license>GPL</metadata_license>
+  <name>Mepo</name>
+  <summary>Fast, simple, and hackable OSM map viewer for Linux</summary>
+  <url type="homepage">https://sr.ht/~mil/mepo</url>
+  <description>
+    <p>
+      Mepo is a fast, simple, and hackable OSM map viewer for
+      Linux. Designed with the Pinephone &amp; mobile linux in mind, works
+      both offline and online.
+    </p>
+  </description>
```

Comment:
this is completely insufficient. Missing `content_rating`, `releases`, `screenshots`, etc.

Reply:
Fixed added more info per your comment- let me know if there's still anything missing